code/scripting/api/libs/controls.cpp

```diff
@@ -314,6 +315,79 @@ ADE_FUNC(AxisInverted, l_Mouse, "number cid, number axis, boolean inverted", "Ge
 		return ADE_RETURN_FALSE;
 }
 
+ADE_VIRTVAR(toggleFlightCursorMode, l_Mouse, "boolean", "Gets or sets whether to use Flight Cursor mode.", "boolean", "true if using Flight Cursor mode, false otherwise")
+{
+	bool choice;
+
+	if (ADE_SETTING_VAR && ade_get_args(L, "*b", &choice))
+	{
+		if (choice)
+			Player_flight_mode = FlightMode::FlightCursor;
+		else
+			Player_flight_mode = FlightMode::ShipLocked;
+	}
+
+	if (Player_flight_mode == FlightMode::FlightCursor)
+		return ADE_RETURN_TRUE;
+	else
+		return ADE_RETURN_FALSE;
+}
```

Comment:
Ah, one more thing, I do plan on having other options for this soon. So we might not want to lock this into a boolean. I don't really know how the lua enums work though... maybe a string name input?

Reply:
Good idea, just updated.